languages/nim/exercises/concept/basics/.meta/example.nim

```diff
@@ -0,0 +1,19 @@
+const expectedMinutesInOven* = 40
+
+const preparationMinutesPerLayer = 2 ##\
```

Comment:
Is there a specific reason for the `##\` at the end of this line?

Reply:
It should be 
```suggestion
const preparationMinutesPerLayer = ## \
```

The purpose is to have the `doc` comment apply to the variable and not make the line ridiculously long.